sdks/cpp/common/examples/full_service/full_service.cpp

```diff
@@ -323,13 +319,14 @@ class CatenaServiceImpl final : public catena::CatenaService::AsyncService {
 
                 case CallStatus::kProcess:
                     new GetValue(service_, dm_, ok);
-                    if (ok) {
+                    if (ok){
                         context_.AsyncNotifyWhenDone(this);
-                        if (validateRequest_() && !context_.IsCancelled()) {
+                        if (validateRequest_()) {
```

Comment:
check this doesn't break the fix for failing to terminate server when a client is connected

Reply:
This branch was made before that fix was merged so this would not have affected it. I updated the branch and checked to be safe, and it is fine.